go.mod

```diff
@@ -2,4 +2,7 @@ module github.com/daixiang0/gci
 
 go 1.14
 
-require golang.org/x/tools v0.0.0-20201118003311-bd56c0adb394
+require (
+	github.com/stretchr/testify v1.7.0 // indirect
```

Comment:
I accept better tests while I want to keep it sample and easy, this introduction increases learning costs for contributors :(